h2o-algos/testMultiNode.sh

```diff
@@ -24,9 +24,9 @@ rm -fr $OUTDIR
 $MKDIR -p $OUTDIR
 
 function cleanup () {
-  kill -9 ${PID_11} ${PID_21} ${PID_31} ${PID_41} ${PID_51} 1> /dev/null 2>&1
-  kill -9 ${PID_12} ${PID_22} ${PID_32} ${PID_42} ${PID_52} 1> /dev/null 2>&1
-  kill -9 ${PID_13} ${PID_23} ${PID_33} ${PID_43} ${PID_53} 1> /dev/null 2>&1
+  kill -15 ${PID_11} ${PID_21} ${PID_31} ${PID_41} ${PID_51} 1> /dev/null 2>&1
```

Comment:
In theory you should send 15 first, wait a little  bit and then send 9. 

Leaving zombies around is pretty dangerous.